SCAutolib/models/gui.py

```diff
@@ -0,0 +1,374 @@
+from SCAutolib import run, logger
+
+import os
+from time import sleep, time
+import cv2
+import numpy as np
+import pytesseract
+import keyboard
+import uinput
```

Comment:
please sort the imports alphabetically per group (as seen in [this guideline document[(https://google.github.io/styleguide/pyguide.html#313-imports-formatting)

Reply:
ok